### ISSUE TITLE
Creating a loadbalancer with private_network_only will result in an error

### ISSUE THREAD

**Issue by h3adex** [stackitcloud/terraform-provider-stackit#462]
Creating a loadbalancer with private_network_only will result in an error. I've used the code supplied in the provider [documentation](https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs/resources/loadbalancer).

Code:
```tf
resource "stackit_loadbalancer" "lb_example_1" {
  project_id = var.project_id
  name       = "lb-example-1"

  ...

  options = {
    private_network_only = false
  }
}
```

![Screenshot 2024-07-10 at 14 37 40](https://github.com/stackitcloud/terraform-provider-stackit/assets/39736813/00814910-91de-411a-a6b9-2f6f9ae65ea0)

**Comment by joaopalet** [comment]
Hello @h3adex,

Thanks for reporting the issue! We will be looking into it soon.

In the meantime, if the API returns `nil` if you set the value to `false` it probably means they are defaulting to `false`. So your configuration would be equivalent to one where the field is not specified.

**Comment by h3adex** [comment]
tyvm

**Comment by joaopalet** [comment]
Hi @h3adex,

This was fixed in [`v0.25.1`](https://github.com/stackitcloud/terraform-provider-stackit/releases/tag/v0.25.1).
